keras_retinanet/bin/train.py

```diff
@@ -45,6 +45,17 @@
 from ..utils.model import freeze as freeze_model
 
 
+def makedirs(path):
+    # Intended behavior: try to create the directory,
+    # pass if the directory exists already, fails otherwise.
+    # Meant for Python 2.7/3.n compatibility.
+    try:
+        os.makedirs(path)
+    except OSError as err_makedirs:
```

Comment:
No need to give the error a name since we're just invoking `raise` without arguments. Other than that, looks good :+1: